docs/pages/guides/mesheryctl/authenticate-to-mesheryctl.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: default
+title: Authenticating to `mesheryctl` 
+permalink: guides/mesheryctl/authenticate-to-mesheryctl
+language: en
+type: Guides
+category: mesheryctl
+---
+
+To authenticate to mesheryctl you have to use the command `mesheryctl system login`. On execution of this command you get a an option to choose a provider, you would have to choose `Meshery` as your provider to authenticate yourself. After doing so, a new window opens up with ways to login. 
+
+## Get your Token
+
+##### There are 2 ways to get your token:
+
+- Get your token through **Meshery Dashboard**, from the `Get Token` option.
+
+    _Downloading the token_
+
+    <a href="{{ site.baseurl }}/assets/img/token/token.png"><img alt="Meshery Dashboard" src="{{ site.baseurl }}/assets/img/token/token.png" /></a>
+    <br/>
+    <br/>
+
+- Get your token through **Meshery CLI**.
+    <br/>
+   > To get the token through `mesheryctl` you would have to use the following command and the path to token for authenticating to Meshery API (default "auth.json").
+    <br/>
+    <pre class="codeblock-pre">
+        <div class="codeblock">
+        mesheryctl system config --token [path-to-file]
+        </div>
+    </pre>
+
+
+## Need for Authentication
```

Comment:
Remove this section and replace with a statement having similar meaning, but that links to the Provider page.